covid_bot/__main__.py

```diff
@@ -67,18 +66,12 @@ def unload(self):
 
     async def on_ready(self):
         await self.wait_until_ready()
-        while True:
-            await bot.change_presence(
-                activity=discord.Activity(
-                    type=discord.ActivityType.watching,
-                    name=f'{len(bot.guilds)} servers | {BOT_SHORT_NAME} help',
-                ),
-            )
-            # Reload the stats Cog every 10 minutes
-            await asyncio.sleep(600)
-            self.unload_extension('covid_bot.cogs.stats')
-            self.load_extension('covid_bot.cogs.stats')
-            logger.info('Reloaded Stats')
```

Comment:
We definitely don't need this anymore. Maybe we'll need it once the stats Cog has more features?